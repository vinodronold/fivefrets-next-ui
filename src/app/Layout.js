import React from 'react'
import { baseFont, fontFamily, color } from './constants/theme'
import HeaderBar from './components/HeaderBar'
import Hamburger from './components/Hamburger'

export default ({ children }) => (
  <div>
    <HeaderBar />
    {children}
    <Hamburger />
    <style jsx global>{`
      html,
      body {
        font-size: ${baseFont};
        font-family: ${fontFamily};
        background: url('/static/img/main_banner.jpg') no-repeat left center fixed;
        background-color: ${color.bg()};
        height: 100%;
        width: 100%;
        padding: 0;
        margin: 0;
        -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
      }
    `}</style>
  </div>
)
