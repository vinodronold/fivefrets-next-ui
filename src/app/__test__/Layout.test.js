import React from 'react'
import renderer from 'react-test-renderer'

import Layout from '../Layout'

describe('Layout.js', () => {
    it('should match Layout Snapshot', () => {
        const _HeaderBar = renderer.create(<Layout />)
        expect(_HeaderBar).toMatchSnapshot()
      })
})