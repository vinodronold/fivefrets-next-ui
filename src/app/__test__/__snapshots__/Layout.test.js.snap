// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout.js should match Layout Snapshot 1`] = `
<div
  data-jsx={694593078}
>
  <div
    data-jsx={1181656448}
  >
    <div
      className="container"
      data-jsx={1181656448}
    >
      <span
        data-jsx={1181656448}
      >
        fivefrets
      </span>
    </div>
  </div>
</div>
`;
